.gitignore

```diff
@@ -52,22 +52,22 @@ dist/
 compile_commands.json
 
 # Example resources
```

Comment:
Sort all these `Example resources`. If you're on VSCode, there is a command to sort lines.